src/test/resources/pages/instructorCourseEditEditInstructorPrivilegesSuccessfulAndCheckEditAgain.html

```diff
@@ -284,7 +284,7 @@
                             </p>
                           </div>
                         </div>
-                        <input name="issectiongroup0set" type="hidden" value="true">
+                        <input name="issectiongroup0set" type="hidden" value="false">
```

Comment:
A portion of the `InstructorCourseEditPageUiTest` had this section changed due to some checkboxes now being deselected by default. Should be able to fix by simulating more clicks.